### PR TITLE
Fix takeOnce

### PR DESCRIPTION
**Pull request by borisenkoandrei**

### CODE REVIEW

File: __tests__/utils/saga.spec.js [hunk 1]
```diff
@@ -122,4 +130,17 @@ describe('utils/saga (takeFirst)', () => {
         expect(refreshOnceControlledHandler).toHaveBeenCalledTimes(2);
     });
 
+    it('takeOnce call twice when action failed', async () => {
+        const requestFailed = createPromise(1, {});
+        store.dispatch(takeOnceFailed(requestFailed.promise));
+        requestFailed.reject();
+        await requestFailed.promise.catch(() => Promise.resolve());
+
```
Comment (ditkachuk) [inline __tests__/utils/saga.spec.js]: check expect error in takeOnceFailed meta

**Comment by ditkachuk** [comment]
bump version in package json to 11

File: __tests__/utils/saga.spec.js [hunk 2]
```diff
@@ -122,4 +130,17 @@ describe('utils/saga (takeFirst)', () => {
         expect(refreshOnceControlledHandler).toHaveBeenCalledTimes(2);
     });
 
+    it('takeOnce call twice when action failed', async () => {
```
Comment (megazoll) [inline __tests__/utils/saga.spec.js]: takeOnce should be callable after error

File: package.json [hunk 3]
```diff
@@ -56,7 +56,6 @@
     "build": "webpack && babel src --out-dir lib",
     "test": "jest",
     "test:watch": "npm test -- --watch",
```
Comment (ditkachuk) [inline package.json]: remove `,`

**Pull request merged**